framework/include/cppmicroservices/ServiceTracker.h

```diff
@@ -478,6 +478,15 @@ namespace cppmicroservices
                             std::shared_ptr<TrackedParamType> const& service) override;
 
       private:
+        /**
+         * Internal method to retrieve a reference following same rules for retrieval
+         * as exported <code>GetServiceReference</code> method
```

Comment:
it would be good to note briefly why this function exists (i.e. why do we need a "no throw" version).